core/templates/dev/head/pages/exploration_editor/editor_tab/StateSolution.js

```diff
@@ -34,21 +34,17 @@ oppia.controller('StateSolution', [
     explorationContextService, angularNameService) {
     $scope.editorContextService = editorContextService;
     $scope.stateSolutionService = stateSolutionService;
-    $scope.isActive = false;
-    $scope.answerCheckPassed = false;
+    $scope.inlineSolutionEditorIsActive = false;
     $scope.INTERACTION_SPECS = INTERACTION_SPECS;
     $scope.interactionHtml = '';
 
     $scope.stateHintsService = stateHintsService;
     $scope.stateSolutionService = stateSolutionService;
     $scope.correctAnswer = null;
 
-    $scope.submitAnswer = function (answer) {
-      $scope.correctAnswer = answer;
-    };
-
-    $scope.toggleIsActive = function() {
-      $scope.isActive = !$scope.isActive;
+    $scope.toggleInlineSolutionEditorIsActive = function() {
```

Comment:
I think it's better to have $scope.openSolutionEditor() and $scope.closeSolutionEditor(), since that's the real point of the action.